extensions/wikia/AdEngine/js/spec/video/playerTracker.spec.js

```diff
@@ -179,14 +169,19 @@ describe('ext.wikia.adEngine.video.player.playerTracker', function () {
 		expect(getTrackedValue('creative_id')).toEqual(92);
 	});
 
-	it('Track data with vulcan data for vulcan ad product', function () {
+	it('Track data with Rubicon data for rubicon ad product', function () {
```

Comment:
I'd like to keep it consistent everywhere. Let's consider changing it once aliases are fixed in prebid.js